### PR TITLE
test: write test for required inputs to read messages

### LINKED ISSUE #2624: form elements: description optimization with `aria-live` annotations
as another optimization we should evaluate to only insert the texts into the related description HTML tags to enable status messages being articulated by screenreaders, with further `role=status` and `role=alert` added to those HTML tags.

_Originally posted by @mfranzke in https://github.com/db-ui/mono/issues/2599#issuecomment-2103907831_

### LINKED ISSUE #2625: form elements
### Which generators are impacted?

- [X] All
- [ ] HTML
- [ ] React
- [ ] Angular
- [ ] Vue
- [ ] Web components
- [ ] Power Apps

### Reproduction case

When switching between the different states of formelements like e.g. a `required` textarea the text content of the invalid HTML element (`.db-textarea:has(textarea:required) .db-infotext`) changes in between todo and the validation message that gets retrieved by the element on invalid state.

### Expected Behaviour

There shouldn't be a reason for the copy to change.

### Screenshots

_No response_

### Browser version

None

### Add any other context about the problem here.

Most likely the solution will be part of https://github.com/db-ui/mono/issues/2624, as we should provide the copy of the message dynamically, instead of switching the HTML elements.

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

closes #2624
closes #2625

Because of the changes in `aria-describedby` the screen reader already announces changes coming from the valid/invalid messages.

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/screen-reader/translations.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	blank: ['Leer'],
+	checked: ['aktiviert'],
+	' of ': [' von '],
+	clickable: ['anklickbar'],
+	'has auto complete': ['mit Auto Vervollständigung'],
+	required: ['erforderlich'],
+	'Please fill out this field..': ['Fülle dieses Feld aus..'],
+	unselected: ['nicht ausgewählt'],
+	selected: ['ausgewählt'],
+	'': ['. Nummernblock eingeschaltet']
```
Comment (mfranzke) [inline showcases/screen-reader/translations.ts:15]: why is this an empty key?

**Review by mfranzke — DISMISSED** [review]

File: showcases/screen-reader/translations.ts (hunk 1 above)
Comment (nmerget) [reply showcases/screen-reader/translations.ts:15]: I guess there is some extra setting for keyboards with a Num block, which isn't the case on the Runner

File: packages/components/src/styles/visually-hidden.scss
Comment (mfranzke) [inline packages/components/src/styles/visually-hidden.scss:1]: what's the difference to the contents out of https://github.com/db-ui/mono/blob/main/packages/foundations/scss/helpers/_a11y.scss#L1-L8 ?

Comment (nmerget) [reply packages/components/src/styles/visually-hidden.scss:1]: fixed

File: packages/foundations/scss/helpers/_a11y.scss [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 	position: absolute !important;
-	white-space: nowrap;
-	inline-size: 1px;
+	inline-size: 1px !important;
+	border-width: 0 !important;
+	border-style: initial !important;
+	border-color: initial !important;
+	border-image: initial !important;
+	padding: 0 !important;
+	pointer-events: none !important;
```
Comment (mfranzke) [inline packages/foundations/scss/helpers/_a11y.scss:16]: @nmerget where have you gotten all of those new properties from? In general a smaller amount should even already do the trick, compare to: https://github.com/h5bp/html5-boilerplate/blob/main/dist/css/style.css#L100-L120

Comment (nmerget) [reply packages/foundations/scss/helpers/_a11y.scss:16]: I looked at https://component.gallery/components/visually-hidden/ and used everyhting from other design systems ^.^

**Review by mfranzke — APPROVED** [review]

**Review by mfranzke — APPROVED** [review]

**Review by mfranzke — APPROVED** [review]

**Pull request merged**